packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from "node-pg-migrate";
+import { stripNewLines } from "../util";
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable(
+    "accounts",
+    {
+      account_id: {
+        type: "UUID",
+        primaryKey: true,
+        comment:
+          "Accounts are undecided and just here for satisfying the future schema",
+      },
+    },
+    {
+      ifNotExists: true,
+    },
+  );
+
+  pgm.createTable(
+    "ids",
+    {
+      version_id: {
+        type: "UUID",
+        primaryKey: true,
+      },
+      id: {
+        type: "UUID",
+        notNull: true,
+        unique: true,
+      },
+    },
+    {
+      ifNotExists: true,
+      comment: stripNewLines(`
+        This table is a boundary to define the actual identification scheme for our kinds of types.
+        Assume that we use the UUIDs on the types to look up more specific ID details.
+        `),
+    },
+  );
+
+  pgm.createTable(
+    "data_types",
+    {
+      version_id: {
+        type: "UUID",
+        primaryKey: true,
```

Comment:
Are we not referencing `ids` yet then?

Reply:
Good spot, have addressed in https://github.com/hashintel/hash/pull/712/commits